### ISSUE TITLE
Get page linked from Apptainer docs?

### ISSUE THREAD

**Issue by klieret** [hsf-training/hsf-training-singularity-webpage#93]

**Comment by mambelli** [comment]
Which Apptainer docs do you mean? Get our training linked in the official Apptainer documentation, like asking for some kind of acknowledgment from them?

**Comment by klieret** [comment]
Yes, I think that was the idea. Not sure how realistic it is, though.

**Comment by mambelli** [comment]
I contacted Dave Dykstra, Apptainer developer. He suggested it would not be good to have that dependency in the docs themselves.  Maybe it would make sense to have a link on the apptainer home page though, once the lesson is out of Beta.  Maybe at https://apptainer.org/getting-started.
Let's get back to them once we promote this out of beta

**Comment by klieret** [comment]
Thanks a lot for reaching out to them!  

I think it would also somewhat limit our options to make this more high energy specific, but perhaps there is still a place for it (like if they had an "external tutorials" section or so)
